tests/testthat/test-ScpModel-Class.R

```diff
@@ -167,64 +171,59 @@ test_that("scpModelResiduals", {
     se <- SummarizedExperiment()
     model <- ScpModel()
     metadata(se)[["test1"]] <- model
-    ## no model = error
-    expect_error(
-        scpModelResiduals(se),
-        regexp = "scpModelFitList.*test1.*scpModelWorkflow"
-    )
     ## No residuals = error
     l <- .createMinimalData(); se <- l$se; a <- l$a
-    model <- .addScpModelFitList(model, rownames(se), dfP = TRUE)
+    colData(se)$condition <- factor(rep(c("A", "B"), each = nrow(se)/2))
+    model <- .addScpModelFitList(model, rownames(se), dfP = TRUE, coefR = TRUE)
     metadata(se)[["test1"]] <- model
+    scpModelInputIndex(se) <- 1
     expect_error(
```

Comment:
Currently the error message indicate the first missing element that is involved in the residuals computation. Should we catch these errors to return a more general error message (`No available scpModelResiduals for model 'model'. Use 'scpModelWorkflow()' to run the scplainer modelling workflow.`) ?